pkg/httpserver/middleware_metric.go

```diff
@@ -10,9 +10,11 @@ import (
 )
 
 const (
-	MetricHttpRequestCount        = "HttpRequestCount"
-	MetricHttpRequestResponseTime = "HttpRequestResponseTime"
-	MetricHttpStatus              = "HttpStatus"
+	detailed                       = "Detailed"
```

Comment:
Do we need to change this to `PerRoute` as well?

Reply:
yap, adjusted that.